contracts/stake-cw20-external-rewards/src/contract.rs

```diff
@@ -829,8 +829,16 @@ mod tests {
         assert_pending_rewards(&mut app, &reward_addr, ADDR2, 2500);
         assert_pending_rewards(&mut app, &reward_addr, ADDR3, 6000);
 
+        // Current height is 1034. ADDR1 is receiving 500 tokens/block
+        // and ADDR2 / ADDR3 are receiving 250.
+        //
+        // At height 101000 99966 additional blocks have passed. So we
+        // expect:
+        //
+        // ADDR1: 5000 + 99966 * 500 = 49,998,000
+        // ADDR2: 2500 + 99966 * 250 = 24,994,000
+        // ADDR3: 6000 + 99966 * 250 = 24,997,500
```

Comment:
I think this comment is totally fine and better than not having one. But ideally we wouldn't need a comment and the actual test code would be more descriptive. 

Something like this:
```
blocks_to_add = 99966
height += blocks_to_add
app.borrow_mut().update_block(|b| b.height = height);

assert_pending_rewards(&mut app, &reward_addr, ADDR1, addr1_total_rewards + blocks_to_add * addr1_per_block_rewards);
...
```

Reply:
Great point. Let's do this as a followup so we can land this.